src/scriptdev2/scripts/eastern_kingdoms/naxxramas/boss_gluth.cpp

```diff
@@ -150,7 +150,9 @@ struct boss_gluthAI : public ScriptedAI
 
                 // Devour a Zombie
                 if (pZombie->IsWithinDistInMap(m_creature, 15.0f))
+                    m_creature->SetFacingToObject(pZombie);
                     m_creature->DealDamage(pZombie, pZombie->GetHealth(), nullptr, DIRECT_DAMAGE, SPELL_SCHOOL_MASK_NORMAL, nullptr, false);
+                    m_creature->SetFacingToObject(m_creature->getVictim());
```

Comment:
The SetFacingToObject() is not required. It won't have any impact on the behavior.
Since the boss it's in combat, it will update the facing to the victim anyway.